files/SMPPSim.sh

```diff
@@ -24,6 +24,10 @@ function die ()
 
 [ -n "${1:-}" ] || die 1 "Missing settings file argument."
 
+# Reset message-id to avoid duplicates in db when restarting the service
+# See 26530
+sed -i -e "s|START_MESSAGE_ID_AT=.*|START_MESSAGE_ID_AT=$(date +%s)|" $1
```

Comment:
Not sure this is the right approach for this. 

1) As this is a docker and is inteded to be used at different places/escenarios. We probable should make this 'behaviour' dependant on an environment variable. Either one actually specifying the value to assign to START_MESSAGE_ID_AT, or a boolean indicating wether to set START_MESSAGE_ID_AT to current timestamp. This way, when launching the docker 

2) There is something we didnt think about, which is a race condition by which:

2.1) smppsim starts with, let's say, START_MESSAGE_ID_AT=1649106822
2.2) an smpp client sends 11+ messages within a short period (say 2 seconds), so current MessageId is now 1649106834
2.3) smppsim crashes and it is restarted by supervisord after 1~2 seconds..
2.4) new smppsim instance ends up with START_MESSAGE_ID_AT=1649106826
2.5) now we may end up (again) with duplicated MessageIds, as client already received IDs for 1649106826..34

So, having this into account, it looks like we should be generating a (long) random number instead of using a timestamp. Something with ¿15? numbers or so.. this probably will ensure we avoid overlapping numbers from fast restarts.